components/vf-masthead/vf-masthead--inlay.hbs

```diff
@@ -1,25 +1,31 @@
-<div class="vf-inlay">
-  <header class="vf-inlay__header">
-    <div class="vf-masthead">
-      <div class="vf-masthead__inner">
-        <div class="vf-masthead__title">
-          <h1 class="vf-masthead__heading">
-            <a class="vf-masthead__heading__link" href="http://dev-vf-theme-prototype.pantheonsite.io">Häring Group</a>
-          </h1>
-        </div>
-      </div>
+<style>
+:root {
```

Comment:
these won't be living here for much longer. I think we'd need a variant for a masthead with images that takes these variables.

And then documentation about theming (which I'm working on, before we cross streams)